ecs-agent/tmds/handlers/fault/v1/handlers/handlers_sudo_integ_test.go

```diff
@@ -0,0 +1,377 @@
+//go:build linux && sudo
+// +build linux,sudo
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package handlers
```

Comment:
Thanks for moving to ecs-agent module. Do we need to update `run-sudo-tests` target to include `ecs-agent` module as well?

https://github.com/aws/amazon-ecs-agent/blob/master/Makefile#L183C6-L184C55

Reply:
Yep it's been added here -> https://github.com/aws/amazon-ecs-agent/pull/4399/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R184

Thanks for double checking

Reply:
Ah right. Not sure how I missed that.